cluster/ci/config/mirroring/origin_v3_11_quay

```diff
@@ -27,11 +27,11 @@ registry.svc.ci.openshift.org/openshift/origin-v3.11:keepalived-ipfailover quay.
 registry.svc.ci.openshift.org/openshift/origin-v3.11:kube-rbac-proxy quay.io/openshift/origin-kube-rbac-proxy:v3.11 quay.io/openshift/origin-kube-rbac-proxy:v3.11.0
 registry.svc.ci.openshift.org/openshift/origin-v3.11:kube-state-metrics quay.io/openshift/origin-kube-state-metrics:v3.11 quay.io/openshift/origin-kube-state-metrics:v3.11.0
 registry.svc.ci.openshift.org/openshift/origin-v3.11:logging-auth-proxy quay.io/openshift/origin-logging-auth-proxy:v3.11 quay.io/openshift/origin-logging-auth-proxy:v3.11.0
-registry.svc.ci.openshift.org/openshift/origin-v3.11:logging-curator quay.io/openshift/origin-logging-curator:v3.11 quay.io/openshift/origin-logging-curator:v3.11.0
-registry.svc.ci.openshift.org/openshift/origin-v3.11:logging-elasticsearch quay.io/openshift/origin-logging-elasticsearch:v3.11 quay.io/openshift/origin-logging-elasticsearch:v3.11.0
+registry.svc.ci.openshift.org/openshift/origin-v3.11:logging-curator5 quay.io/openshift/origin-logging-curator5:v3.11 quay.io/openshift/origin-logging-curator5:v3.11.0
```

Comment:
Don't update these files in your PR, I have to manage them

Reply:
I'm dropping cluster/ci/config/mirroring/origin_v3_11 and cluster/ci/config/mirroring/origin_v3_11_quay